collector/consts.go

```diff
@@ -19,12 +19,8 @@ const (
 )
 
 var (
-	// blxDefaultURL options - via https://docs.bloxroute.com/introduction/cloud-api-ips
-	// wss://virginia.eth.blxrbdn.com/ws
-	// wss://uk.eth.blxrbdn.com/ws
-	// wss://singapore.eth.blxrbdn.com/ws
-	// wss://germany.eth.blxrbdn.com/ws
-	blxDefaultURL = common.GetEnv("BLX_URI", "wss://virginia.eth.blxrbdn.com/ws")
+	// Bloxroute URL - should point to local Gateway GRPC port
+	blxDefaultURL = common.GetEnv("BLX_URI", "127.0.0.1:1001")
```

Comment:
how do we deal with having this code usable by everyone with an API key? i.e. how do we manage compatibility with both grpc and websocket subscriptions?

i think we want the code be able to do both, could distinguish based on `wss://` prefix 🤔